21/streams/core-concepts.html

```diff
@@ -224,7 +224,7 @@ <h3><a id="streams_out_of_ordering" href="#streams_out_of_ordering">Out-of-Order
 
     <p>
         Besides the guarantee that each record will be processed exactly-once, another issue that many stream processing application will face is how to
-        handle <a href="tbd">out-of-order data</a> that may impact their business logic. In Kafka Streams, there are two causes that could potentially
+        handle <a href="https://www.confluent.io/wp-content/uploads/streams-tables-two-sides-same-coin.pdf">out-of-order data</a> that may impact their business logic. In Kafka Streams, there are two causes that could potentially
```

Comment:
We should not link to CP webpage from AK. Also, I am wondering if linking to the paper was the goal? If yes, we can also link to https://dl.acm.org/citation.cfm?id=3242155